app/src/main/java/com/example/mycalculator/mvp/CalculatorModelImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.example.mycalculator.mvp;
+
+public class CalculatorModelImpl implements CalculatorContract.calculatorModel {
+
+    public String getResult;
+    private String operatorOne;
```

Comment:
no quedaria mejor ponerle firstOperator & secondOperator

Reply:
sure!

Reply:
Done. You were right thx!